cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaRoller.java

```diff
@@ -424,10 +435,10 @@ private boolean podWaitingBecauseOfAnyReasons(Pod pod, Set<String> reasons) {
     }
 
     private boolean isPending(Pod pod) {
```

Comment:
Should this still be called `isPending`? It seems it's more like `isPendingAndUnschedulable`